### DISCUSSION TITLE
How to change the locale of `filterVariant: 'date-range'`

### DISCUSSION THREAD

**Discussion by XindiShang** [KevinVandy/mantine-react-table#103] [Q&A] [answered]
would appreciate it if anyone can help me 🙏

**Comment by KevinVandy** [comment, marked as answer]
There is a `locale` prop you can specify in `mantineFilterDateInputProps` table option or column option

https://mantine.dev/dates/date-picker/#localization

**Reply by XindiShang** [reply]
Thanks 👏
